vignettes/deploying-dccvalidator.Rmd

```diff
@@ -22,8 +22,8 @@ please read the
 The app is deployed on the server through the following steps:
 
 1. ssh into the Shiny Pro server and navigate to `/home/kwoo/ShinyApps/dccvalidator-app`
-1. `git pull` changes from GitHub
-1. To ensure packages are up-to-date, open an R session and run `renv::restore()`
+2. `git pull` changes from GitHub
```

Comment:
FYI markdown will automatically increment the numbers in an ordered list. This is how the [rendered page](https://sage-bionetworks.github.io/dccvalidator/articles/deploying-dccvalidator.html) appears, even though we are currently using `1., 1., 1.,`
<img width="649" alt="Screen Shot 2019-11-05 at 11 55 34 AM" src="https://user-images.githubusercontent.com/4452678/68241283-430ad600-ffc3-11e9-9d92-2ef440bddc01.png">